LEVEL 1/terms.py

```diff
@@ -0,0 +1,47 @@
+# 코딩테스트 연습 / 2023 KAKAO BLIND RECRUITMENT / 개인정보 수집 유효기간
+# 다른풀이로는 한달이 28일로 고정되어있다는 점을 이용하여 일 수 비교로 풀 수 있다.
+
+def solution(today, terms, privacies):
+    answer = []
+    # 오늘의 날짜
+    t_y, t_m, t_d = map(int, today.split('.'))
+    
+    # { name : period }로 계약 저장
+    terms_dict = {}
+    for obj in terms:
+        name, period = obj.split()
+        terms_dict[name] = int(period)
+    
+    # privacies 반복문으로 비교 시작
+    for i in range(len(privacies)):
```

Comment:
```suggestion
    for i in range(len(privacies)):
```
파이썬에서 for문을 사용 할때,, range로 index로 다루기 보다는 가능하면 `for privacy in privacies:` or `for index, privacy in enumerate(privacies): `를 이용하면 좋을 듯!!

Reply:
enumerate 활용을 해봐야겠구만. 땡큐~